### DISCUSSION TITLE
Installing the IDE without admin rights

### DISCUSSION THREAD

**Discussion by steve-itb** [posit-dev/positron#3694] [Q&A]
Hi,

I tried to install this on my work laptop (windows) but it requires an admin to do it.
Is it possible to install on this release on windows without admin rights?

Thank you for your help

**Comment by cjreiner** [comment]
Related note, installing on a corporate computer (windows) that allows elevated permissions (BeyondTrust) can trigger A LOT of warnings, (Registry Edits and Command Processor). I realize this is generic feedback, but installing RStudio or VSCode on the same machine does not triggers these warnings. I think this is important to minimize so corporate user can more easily test the IDE.

Positron Version: 2024.06.1 (system setup) build 2024.06.1-27
Code - OSS Version: 1.90.0
Commit: a893e5b282612ccb2200102957ac38d3c14e5196
Date: 2024-06-26T01:33:58.809Z
Electron: 29.4.0
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by olivroy** [comment]
I opened this as a FR in #3703
